examples/teams/main.tf

```diff
@@ -45,3 +45,8 @@ output "memberships" {
   description = "A map of members and admins keyed by username."
   value       = module.members.memberships
 }
+
+output "teams" {
+  description = "A map teams with members and corresponding roles."
```

Comment:
Perhaps better worded as "a map of teams with...". WDYT?

Reply:
fixed